examples/dao/setup_sigma_dao.sql

```diff
@@ -61,3 +68,9 @@ EXECUTE FUNCTION sigmadao_trigger_fn();
 
 -- grant privileges to relation sigma_daos
 GRANT ALL PRIVILEGES ON TABLE sigma_daos TO algorand;
+
+-- Function to search dao name in sigma_daos relation
+CREATE OR REPLACE FUNCTION search_sigma_daos(daoToBeSearched TEXT)
+RETURNS SETOF sigma_daos AS $$
+	SELECT * FROM sigma_daos WHERE dao_name ilike ('%' || daoToBeSearched || '%');
+$$ LANGUAGE SQL STABLE;
```

Comment:
why do we need this function? We can use SELECT directly?

Reply:
This is needed because postgraphile does only the exact search. E.g if there a dao named 'DAO' then if you search for 'dao' or 'DA', then no result will be returned.

Reply:
My question was about introducing the function rather than using `select` directly. I thought we can add any handler to postgraphile, including raw SQL.

Reply:
We have to create function in postgresql so that postgraphile can recognize it as query. For more info on custom query: https://www.graphile.org/postgraphile/custom-queries/

Reply:
OK, thanks!